examples/assets/ui/example.ron

```diff
@@ -61,6 +61,31 @@ Container(
             ),
             image: PartialTexture(File("texture/test_texture.png", ("IMAGE", ())),0.34,0.0,1.0,0.333),
         ),
+        //9-Slice
+        Image(
+            transform: (
+                id: "Partial Texture Logo",
+                x: -250.,
+                y: -40.,
+                width: 150.,
+                height:  80.,
+                tab_order: 1,
+                anchor: TopRight,
+                mouse_reactive: true,
+            ),
+            image: NineSlice(
+                    x_start:0,
```

Comment:
nit: could you use 4 space indent and add a space after each `:`

Reply:
I believe I fixed this